### PR TITLE
Blue button

### PR DESCRIPTION
**Pull request by windresistance**

### CODE REVIEW

File: views/pages/home.php [hunk 1]
```diff
@@ -86,11 +86,16 @@
...
               エアコン室外機カバー<br>
               商品詳細はこちら
             </button>
           </div>
+          <br>
+          <button class="blue-button">
+            エアコン室外機カバー<br>
+            商品詳細はこちら
+          </button>
         </div>
```
Comment (antoine-boue) [inline views/pages/home.php:99]: Hmm I dont see this button on the design ?

Comment (windresistance) [reply views/pages/home.php:99]: It's on the product page, on the left nav

Comment (windresistance) [reply views/pages/home.php:99]: Ah sorry I meant to remove it from the home page, let me do that

Comment (antoine-boue) [reply views/pages/home.php]: Ah ok, no problem

**Review by antoine-boue — APPROVED** [review]

**Pull request merged**